content/articles/password-cracking-with-john-the-ripper/index.md

```diff
@@ -0,0 +1,168 @@
+
+```bash
+$ make
+```
+Lastly run the `make install` command to install John the Ripper.
+
+```bash
+$ make install
+```
+![Run make install command](/engineering-education/password-cracking-with-john-the-ripper/make_install.png)
+
+Run the commands below to see if the installation was successful.
+```bash
+$ cd ..
+```
+```bash
+$ cd run
+```
+```bash
+$ ./john
+```
+![Confirm Installation](/engineering-education/password-cracking-with-john-the-ripper/run.png)
+
+#### 2. Installing from the package
+You can also install John the Ripper by running the command below:
+
+```bash
+$ sudo apt install john
+```
+
+![Install from package](/engineering-education/password-cracking-with-john-the-ripper/installing.png)
+
+### How John the Ripper cracks passwords
+During the cracking process, John the Ripper uses a rainbow table approach where it takes words from an in-built dictionary that comes with it.
+
+It then compiles the variations of that dictionary and compares the hashed password to what is in the password file trying to find a match. This is repeated until a match is found.
+
+### Example cases of cracking passwords
+You will need to run different commands depending on the type of the target you are exploiting.
+Let's look at some instances we might come across when cracking passwords using John the Ripper.
+
+#### 1. Cracking a `zip/rar` password-protected file
+
+Cracking a `zip` or `rar` file password is done using the same approach.
+
+First, we will need to access the hash of the password we are going to crack.
+Note the name of your zip file as you will be required to state it in the rest of the commands.
+
+To access the password hash, navigate to the location of your zip password-protected file and run the command below in the terminal:
+```bash
+$ zip2john Test.zip
+```
+
+You can export the output to a text document to save the password hash since we are going to use it later.
+```bash
+$ zip2john Test.zip > hash.txt 
+```
+
+To begin the attack on your zip file, run the command below:
+```bash
+$ john --format=zip hash.txt
+```
+
+In the command above, we have specified the format of the target file.
+
+The password cracking process is initiated. This is where the strength of the password comes in. The stronger the password, the more the time taken to perform an attack.
+
+After a successful attack, the password will be displayed on the terminal.
+
+#### 2. Cracking a user account password in Kali Linux
+
+Moving on, we will learn how to crack another user account's password using John the Ripper.
+
+First, let's create another user account that we are going to crack its password.
+Run the command below in the terminal.
+```bash
+$ useradd -r James
+```
+
+We have created a user account named `James`.
+
+Now, let's set the password for the account.
+```bash
+$ passwd James
+```
+
+Run the command below to crack James' password.
+```bash
+$ john /etc/shadow
+```
```

Comment:
These commands require `sudo` or `root` user permissions to work. Consider using the `sudo` prefix in the highlighted commands and add more information on the `passwd` command to make it clear.

I've also noticed that some images are not used. Please delete them.